frontend/src/features/meta-info/loader/Loader.module.css

```diff
@@ -0,0 +1,12 @@
+.loaderContainer {
+    height: 100vh;
+    position: absolute;
+    justify-content: space-around;
+    align-items: center;
+    top: 0;
```

Comment:
You can add a semi-transparent background here so that your loader is clearly visible over the content.